api/types/installers/installer.sh.tmpl

```diff
@@ -38,22 +44,38 @@
     echo "Unsupported distro: $ID"
     exit 1
   fi
+  
+  if on_azure ; then
```

Comment:
Can we explicitly check for the "cloud provider". Currently this script assume that "it not azure then ec2" which may not be true when someone makes a typo. I'd change the `else` to `elif on_aws` and add `else exit 1;` to account for that.